waspc/data/Generator/templates/sdk/wasp/auth/session.ts

```diff
@@ -69,7 +69,7 @@ async function getAuthUserData(userId: {= userEntityUpper =}['id']): Promise<Aut
     throwInvalidCredentialsError()
   }
 
-  return createAuthUserData(user);
+  return createAuthUserData(user!);
```

Comment:
Again, I'm not sure why this is needed since we have the check `if (!user) {`. Maybe it's not obvious to the compiler that the `throwInvalidCredentialsError` fn throws?